README.rst

```diff
@@ -97,6 +97,14 @@ For instance, ``has_attrs(name="bob")`` is equivalent to ``has_attrs(name=equal_
       # Matches ["a", "b"], ["b", "a"] and ["a", "c", "b"]
       # but not ["a", "c"] nor ["a"]
 
+* ``contains_only(matcher)``: matches an iterable if every element matches `matcher`.
+  For instance:
+
+  .. code:: python
+
+      assert_that(result, contains_only(has_attrs(name=is_a(str)))
+      # matches all objects with a string name attribute
```

Comment:
The other comments explaining what matches use concrete examples, in the optimistic hope that that's clearer than describing in the abstract. Could this be rewritten to match the style of the other examples?